### DISCUSSION TITLE
404 returned when calling observations/latest for a valid station ID

### DISCUSSION THREAD

**Discussion by shampeon** [weather-gov/api#294] [Q&A]
**NOTE**
Please do not report operational issues (such as outages) here. We can not support these issues via this forum. Please report these as described here: https://weather-gov.github.io/api/reporting-issues

**Describe the bug**

I'm unable to get the latest observations for the C4253 station as it results in a 404 error.

The station appears to be reporting current conditions on the web site:
https://forecast.weather.gov/MapClick.php?lon=-120.04537582397462&lat=38.47292388552813

**To Reproduce**

The following call results in a 404:
https://api.weather.gov/stations/C4253/observations/latest

with a correlationId: 9675445d

It's a valid station:
https://api.weather.gov/stations/C4253

The features array is empty when calling:
https://api.weather.gov/stations/C4253/observations

**Expected behavior**

The current observations for the station.

**Screenshots**
n/a

**Environment**

This occurs when I call in a browser and when using cURL.

**Comment by ivanoats** [comment]
I'm seeing a similar problem on https://api.weather.gov/stations/wpow1/observations/latest

**Comment by scadergit** [comment]
This is interesting, because Mesowest reports C4253 hasn't reported an observation since 8/13:

![image](https://user-images.githubusercontent.com/17500494/91778288-9df76480-ebb7-11ea-90cc-8ce7a0609038.png)

https://mesowest.utah.edu/cgi-bin/droman/past_dyn.cgi?stn=C4253&hour1=16&day1=31&month1=8&year1=2020&product=&time=LOCAL&unit=0&order=1

WPOW1 does have observations, but the API is throwing a 500 because it doesn't like something in the data it's reporting. I would open an operational ticket with the helpdesk on that one.

**Comment by StephenClouse** [comment]
![image](https://user-images.githubusercontent.com/468025/91778946-b10b3400-ebb9-11ea-97de-2011eda15296.png)
